### PR TITLE
uncommented .atwho(at_config)

### LINKED ISSUE #9821: Turn on username autocomplete in rich editor
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file `app/views/editor/rich.html.erb` in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -538,7 +538,7 @@
     $('.wk-wysiwyg')
         .atwho(hashtags_config)
         .atwho(emojis_config)
-        //.atwho(at_config) // off for performance reasons for now
+        .atwho(at_config)
   })
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by BajajSmriti**
Fixes #9821

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by BajajSmriti** [comment]
@publiclab/reviewers @TildaDares Please review.

File: app/views/editor/rich.html.erb [hunk 1]
```diff
@@ -538,7 +538,7 @@
     $('.wk-wysiwyg')
         .atwho(hashtags_config)
         .atwho(emojis_config)
-        //.atwho(at_config) // off for performance reasons for now
+        .atwho(at_config) // off for performance reasons for now
```
Comment (TildaDares) [inline app/views/editor/rich.html.erb:541]: @BajajSmriti Can you remove the comments? Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/editor/rich.html.erb (hunk 1 above)
Comment (jywarren) [reply app/views/editor/rich.html.erb:541]:
```suggestion
        .atwho(at_config)
```

**Review by TildaDares — APPROVED** [review]
LGTM🎉

**Comment by jywarren** [comment]
Oh hmm @BajajSmriti @TildaDares can you look what went wrong in the system tests? Thank you! Happy to help if it's confusing!

**Comment by TildaDares** [comment]
@jywarren I'm working on it now. That particular test has been failing before my PR.

**Comment by jywarren** [comment]
Ah, super. thank you!! @TildaDares if you could open an issue for it that'd be great! Thanks, everyone!!!

**Comment by jywarren** [comment]
@TildaDares @BajajSmriti would you mind testing this out on https://stable.publiclab.org once it builds? Thank you!!!

**Comment by TildaDares** [comment]
@jywarren It returned this error
``` 
Failed to load resource: the server responded with a status of 500 (Internal Server Error) https://stable.publiclab.org/api/srch/profiles?query=&sort_by=recent&field=username
```

I had the same issue when I was testing the username autocomplete on stable even though it works perfectly on unstable. I tried accessing the data manually by using the url but it takes me to an error page. 

Removing the `sort_by` field from the query works https://stable.publiclab.org/api/srch/profiles?query=a&field=username.

**Pull request merged**